src/components/utils/useListNavigation/useListNavigation.ts

```diff
@@ -38,8 +38,16 @@ export function useListNavigation<ItemType, AnchorType extends HTMLElement>({
         }
     }, [items, reset]);
 
-    useLayoutEffect(() => {
+    const canNavigate = useMemo(() => {
```

Comment:
I guess we can avoid memoization for boolean value

Reply:
I thought we don't need to calculate skip for every element on every render, since there can be a lot of items, but you are right, it is a rare case that the available element will not be at the beginning

Reply:
Yeah, but useMemo is not free to spend it for booleans